helix/principles/subsystems/commerce.rst

```diff
@@ -0,0 +1,53 @@
+Sitecore Experience Commerce
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When working with Sitecore Experience Commerce, there are several
+more subsystems that you could end up posting code or config changes
+to. These include:
+
+- Sitecore Commerce Engine roles (Shops, Authoring, Ops, Minions)
+- Sitecore Commerce Business Tools
+- Identity Server
+
+This does add complexity to your solution & deployment - both locally
+and to your Production & Non-Production environments. The key thing is
+to remember that this added complexity shouldn't stop you from following
+all of the Helix guidelines discussed in this document. The Common Closure 
+Principal states that Classes that change together are packaged together,
+and that still applies to your Commerce functionality. 
+
+The commerce functionality will be split up between the Modules you have 
+throughout your Foundation, Feature & Project layers.
+
+Module Structure
+^^^^^^^^^^^^^^^^
+Not all of your modules will contain functionality that runs on both the
+Website & in the Commerce Engine, but it is very common. For example a Product
+Compare module that stores the list of products in the Engine, and displays this
+list to the end user on the Website. In this scenario the functionality will
+be split between the Engine & the Website - however as these two element are very

```

Comment:
grammar check: "however, as these two _elements_ are very coherent, the Common Closure Principal states that _they_ should live together."